miden-lib/asm/sat/asset.masm

```diff
@@ -0,0 +1,108 @@
+use.miden::sat::account
+
+# CONSTANTS
+# =================================================================================================
+
+const.FUNGIBLE_ASSET_MAX_AMOUNT=9223372036854775807
```

Comment:
I wonder if this should actually be $2^{63} - 1$.

Reply:
Yes, this is the result I got:
```
>>> (2**63) - 1
9223372036854775807
```
 
Did you get a different value?

Reply:
Ah - you are right!